pallets/committee/src/lib.rs

```diff
@@ -357,7 +355,11 @@ pub mod pallet {
             vote: Vote,
         ) -> DispatchResultWithPostInfo {
             // Only members can vote
-            let voter = Self::ensure_member(origin)?;
+            let caller = T::ProposalVoteOrigin::ensure_origin(origin)?;
+            let voter = CommitteeMember::new(
+                caller.clone(),
+                <Members<T>>::get(caller).ok_or(<Error<T>>::NotMember)?,
+            );
```

Comment:
Since only members are allowed to vote, I believe we should only check for inclusion in `Members` instead adding another origin, which will probably do the same as `ensure_member`

Reply:
ur right LOL, added an origin for Vote creates a lot of trouble for me QAQ

But the logic of `EnsureMember` contains the logic of `ensure_member`, so I want to remove `ensure_member` crazily LOL

Reply:
> BTW, I'm not afraid of Origin anymore now hhh

That's awesome !